osu.Game/Screens/Play/HUD/DefaultKeyCounterDisplay.cs

```diff
@@ -36,13 +36,19 @@ protected override void Update()
             Size = keyFlow.Size;
         }
 
-        public override void Add(InputTrigger trigger) =>
-            keyFlow.Add(new DefaultKeyCounter(trigger)
+        protected override KeyCounter CreateCounter(InputTrigger trigger)
+        {
+            var keyCounter = new DefaultKeyCounter(trigger)
             {
                 FadeTime = key_fade_time,
                 KeyDownTextColor = KeyDownTextColor,
-                KeyUpTextColor = KeyUpTextColor,
-            });
+                KeyUpTextColor = KeyUpTextColor
+            };
+
+            keyFlow.Add(keyCounter);
```

Comment:
This should not be in a `Create` method.

Reply:
I mean I agree, but a few renames (`Create` -> `AddInternal`) would have made this passable I think...